components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -1305,6 +1310,20 @@ export function createWalletApi () {
       //
       // Transactions
       //
+      getFVMAddress: query<Map<string, {address: string, fvmAddress: string | null}>, GetFVMAddressArg>({
+        queryFn: async (arg, api, extraOptions, baseQuery) => {
+          try {
+            const { braveWalletService } = baseQuery(undefined).data
+            return {
+              data: (await braveWalletService.convertFEVMToFVMAddress(arg.isMainNet, arg.addresses)).result
```

Comment:
Please split this into a separate variable, so you can cast the `null` into `undefined`.

Reply:
Actually iOS doesn't support map<string, string?> in Mojo, so i've switched to map<string, string>